Makefile

```diff
@@ -101,6 +101,11 @@ DISABLE_TLS := $(or ${DISABLE_TLS},false)
 ENABLE_ORG_TENANCY := $(or ${ENABLE_ORG_TENANCY},False)
 ALLOW_CONVERGED_FLOW := $(or ${ALLOW_CONVERGED_FLOW}, false)
 ENABLE_ORG_BASED_FEATURE_GATES := $(or ${ENABLE_ORG_BASED_FEATURE_GATES},False)
+OPENSHIFT_MAJOR_VERSION := $(or ${OPENSHIFT_MAJOR_VERSION}, 4)
+OPENSHIFT_RELEASE_API_BASE_URL := $(or ${OPENSHIFT_RELEASE_API_BASE_URL}, https://api.openshift.com/api/upgrades_info/v1/graph)
+OPENSHIFT_SUPPORT_LEVEL_API_BASE_URL := $(or ${OPENSHIFT_SUPPORT_LEVEL_API_BASE_URL}, https://access.redhat.com/product-life-cycles/api/v1/products)
```

Comment:
We already have the default value in `internal/releasesources/config.go`. Any reason to have it here as well?

Reply:
We plug these variables as options for `./tools/deploy_assisted_installer_configmap.py` script, which is triggered by test-infra with `skipper make ... deploy-all ...` in the process of deployment of `assisted-service`, so need to give defaults in case it is nut given for this env vars, not sure how `internal/releasesources/config.go` defaults can be used

Reply:
No reason I guess, I was following the pattern, I will remove it thanks

Reply:
OK, thinking about this again, I need these defaults because of the way we currently deploy the `configmap`, We specify the options, so a value must be given, therefore I need to give it the default